res/xml/bone.xml

```diff
@@ -0,0 +1,45 @@
+<?xml	version="1.0"	encoding="utf-8"?>
+<keyboard>
+	<row>
+		<key   width="1"   key0="j"							key2="°"						key4="esc"/>
+		<key   width="1"   key0="d"							key2="§"						key4="_"/>
+		<key   width="1"   key0="u"							key2="ℓ"						key4="["/>
+		<key   width="1"   key0="a"							key2="»"						key4="]"/>
+		<key   width="1"   key0="x"							key2="«"						key4="^"/>
```

Comment:
`«` and `»` are `Fn+<` and `Fn+>`. Are there really needed on the main pane ?

Reply:
ill have a look into this later